src/components/clipboard-card/clipboard-card.html

```diff
@@ -9,18 +9,21 @@
     {{payProDetailsData?.host}}
   </div>
   <ion-note *ngIf="payProDetailsData && !payProDetailsData.error" item-right>
-    <div class="amount">
+    <div *ngIf="!payProDetailsData?.isFiat" class="amount">
```

Comment:
On invoice flow, the currency is not set so we display invoiceData.currency until buyerSelectedCurrency is set.

On PayPro flow, isFiat will not be set so it will be undefined as default.

payProDetailsData.isFiat is set here: https://github.com/bitpay/copay/blob/2dc09a5466d2edd0a0c226cb37d7286d1e8a8506/src/pages/home/home.ts#L368